WordPress/Classes/WordPress.xcdatamodeld/WordPress 31.xcdatamodel/contents

```diff
@@ -248,6 +248,7 @@
         <attribute name="title" optional="YES" attributeType="String">
             <userInfo/>
         </attribute>
+        <attribute name="videopressGUID" optional="YES" attributeType="String" syncable="YES"/>
```

Comment:
Modifying `WordPress 31.xcdatamodel` should be fine as we haven't done an internal build yet.